frontend/src/components/Patchnotes.jsx

```diff
@@ -22,6 +22,44 @@ const Patchnotes = ({ game }) => {
   const [error, setError] = useState(null);
   const [modalOpen, setModalOpen] = useState(false);
   const [currentReply, setCurrentReply] = useState({ message: '', replyToId: null, commentId: null, parentReplyId: null });
+  const [filter, setFilter] = useState('all');
+
+  const keywords = {
+    "projectile size": { nerf: ["reduced", "smaller"], buff: ["increased", "larger"] },
+    "damage": { nerf: ["decreased"], buff: ["increased"] },
+    "self-damage": { nerf: ["increased"], buff: ["reduced"] },
+    "health": { nerf: ["decreased"], buff: ["increased"] },
+    "armor": { nerf: ["decreased"], buff: ["increased"] },
+    "shields": { nerf: ["decreased"], buff: ["increased"] },
+    "radius": { nerf: ["decreased", "reduced"], buff: ["increased"] },
+    "width": { nerf: ["decreased"], buff: ["increased"] },
+    "height": { nerf: ["decreased", "reduced"], buff: ["increased"] },
+    "cooldown": { nerf: ["increased"], buff: ["decreased"] },
+    "range": { nerf: ["reduced"], buff: ["increased"] },
+    "recovery": { nerf: ["increased"], buff: ["decreased"] },
+    "speed": { nerf: ["decreased"], buff: ["increased"] },
+    "knockback": { nerf: ["reduced", "decreased"], buff: ["increased"] },
+    "duration": { nerf: ["increased"], buff: ["decreased"] },
```

Comment:
Is this strictly true? I assume you are thinking about like debuffs or cooldowns or something, but what if a character has like a shield with a duration and the longer duration is actually a buff?